code/game/machinery/bodyscanner_console.dm

```diff
@@ -36,7 +35,7 @@
 
 /obj/machinery/body_scanconsole/proc/FindDisplays()
 	for(var/obj/machinery/body_scan_display/D in SSmachines.machinery)
-		if(D.tag in display_tags)
+		if(D.id_tag == connected.id_tag)
```

Comment:
```suggestion
		if(D.id_tag in display_tags)
```
I believe the intent of this was so that you could have a many-to-one connection? I don't fully understand the reason for removing this.

Reply:
This list doesn't seem to get populated anywhere. Multiple displays should still connect if they have same tag as the connected scanner if you mean that.

Reply:
As far as I get it, previous implementation attempted to use default `/datum/var/tag` property (acc. to DM reference, to be used with `locate()`) on the displays themselves, which wasn't set and thus not working neither on ministation nor downstream. The ministation2 map sets `id_tag` property (as defined in `/obj/machinery` IIRC) instead, which ostensibly fits the usecase better: setting it to the same value on a bodyscanner and multiple displays would likewise achieve the many-to-one connection.

Reply:
> This list doesn't seem to get populated anywhere. Multiple displays should still connect if they have same tag as the connected scanner if you mean that.

It's supposed to be set by mappers.

> setting it to the same value on a bodyscanner and multiple displays would likewise achieve the many-to-one connection.

I suppose that *is* true. We can probably just revisit it if id_tag on body scanner consoles gets any other uses that would cause problems if shared by multiple consoles.